server/controller/src/main/java/ai/starwhale/mlops/domain/dataset/DatasetService.java

```diff
@@ -464,14 +464,24 @@ public void build(CreateBuildRecordRequest request) {
         if (res) {
             // start build
             var user = userService.currentUserDetail();
-            DockerImage image;
-            var runtimeProperties = systemSettingService.getRunTimeProperties();
-            if (runtimeProperties != null && runtimeProperties.getDatasetBuild() != null
-                    && StringUtils.hasText(runtimeProperties.getDatasetBuild().getImage())) {
-                image = new DockerImage(systemSettingService.getRunTimeProperties().getDatasetBuild().getImage());
-            } else {
-                image = new DockerImage("docker-registry.starwhale.cn/star-whale/starwhale:latest");
+            DockerImage image = new DockerImage("docker-registry.starwhale.cn/star-whale/base:latest");
+            String swVersion = "0.5.6";
```

Comment:
is it possible to make this configurable in system setting

Reply:
done